server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -152,6 +156,9 @@ public class CompactionCoordinator implements CompactionCoordinatorService.Iface
   private static final Cache<ExternalCompactionId,RunningCompaction> COMPLETED =
       Caffeine.newBuilder().maximumSize(200).expireAfterWrite(10, TimeUnit.MINUTES).build();
 
+  private static final Cache<Path,Integer> CHECKED_TABLET_DIRS =
```

Comment:
Why make this static?

Reply:
I could remove the static modifier, but I'm not sure that it really matters. There is only one instance of this class running inside the Manager.

Reply:
If there is no need for it to be static, the making it an instance var feels a bit cleaner.  Making it an instance variable makes less assumptions about how other code uses the class and it makes some test easier to implement (like mocking or overriding the cache in unit test).

Reply:
Removed static modifiers from new code in 27cc81e